presto-main/src/main/java/io/prestosql/execution/SqlQueryManager.java

```diff
@@ -313,4 +323,21 @@ private void enforceCpuLimits()
             }
         }
     }
+
+
+    /**
+     * Enforce query scan bytes limits
+     */
+    private void enforceScanLimits()
+    {
+        for (QueryExecution query : queryTracker.getAllQueries()) {
```

Comment:
You can make this function similar to https://github.com/prestosql/presto/blob/0050a9801f85e3411a4bb74fc0e8ea6ead50d41c/presto-main/src/main/java/io/prestosql/execution/SqlQueryManager.java#L335 just use `query.getQueryInfo().getQueryStats().getPhysicalInputDataSize()` instead.

Reply:
As mentioned by @losipiuk in https://github.com/prestosql/presto/pull/2083#discussion_r430230345  , there may be performance issues with getQueryInfo. And physicalInputDataSize is not in BasicQueryInfo. should I add physicalInputDataSize to BasicQueryInfo, or add new functions to get physicalInputDataSize.
```
losipiuk 29 days ago Member
I am a bit concerned about performance impact from frequent calling to getQueryInfo(). Can you use more lightweight getBasicQueryInfo instead?
```

Reply:
@raunaqmorarka Can you take a look at this?

Reply:
We can add `physicalInputDataSize` to BasicQueryStats and switch to using `query.getBasicQueryInfo().getBasicQueryStats().getPhysicalInputDataSize()`

Reply:
Done